packages/admin-stories/src/docs/components/FinalFormRangeInput/FinalFormRangeInput.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/Final Form Range Input" />
```

Comment:
the name of the component is `FinalFormRangeInput`, I don't see a reason for those spaces